### DISCUSSION TITLE
Stop timelapse problem

### DISCUSSION THREAD

**Discussion by Reflexeye** [gopro/labs#884] [Q&A]
No problems starting a timelapse with solar time (e.g. start 60m before sunrise at a particular longitude and lat).
But - stopping is proving elusive. - the additional code I've put in  using the QR app for ending 60m after sunset:
!s3600E*LLTZ=latitude,longitude,1 (1=summertime UK)
Any help would be great.

**Comment by dnewman-gpsw** [comment]
Don't combine *LLTZ With your command, LLTZ is a set once for you location thing.

Always list you camera model and firmware version. 

```!s-60S!s60E``` start minute before, end a minute after, is working correctly on HERO12.

**Reply by Reflexeye** [reply]
Hero12 fw 02.20.70
Therefore !r-3600S!s3600E (start 1hr before rise, end 1hr after set) should work?

**Reply by dnewman-gpsw** [reply]
Yes, if the camera boot into the default mode (timelapse / nightlapse etc.)

**Comment by Reflexeye** [comment]
You mean preferences/general/default preset/last used timelapse ?

**Reply by dnewman-gpsw** [reply]
correct.
